File: spring/prog_terminal_marittimo/src/main/java/progetto_terminal_marittimo/porto.java
package progetto_terminal_marittimo;

public class porto {
    private int ID;
    private String nome;
    private String nazionalità;


    public porto(int id, String n, String naz){
        this.ID=id;
        this.nome=n;
        this.nazionalità=naz;
    }

    public int getID(){
        return this.ID;
    }

    public String getnome(){
        return this.nome;
    }

    public String getNazionalità(){
        return this.nazionalità;
    }
}

File: pagine/js/porto.js
class Porto {
    id;
    nome;
    nazionalita;
  
    constructor(id, n, naz) {
      this.id = id;
      this.nome = n;
      this.nazionalita = naz;
    }
  
    getID() {
      return this.id;
    }
  
    getNome() {
      return this.nome;
    }
  
    getNazionalita() {
      return this.nazionalita;
    }
  }
  